packages/client/tests/functional/fulltext-search/tests.ts

```diff
@@ -0,0 +1,158 @@
+import testMatrix from './_matrix'
+
+// @ts-ignore this is just for type checks
+declare let prisma: import('@prisma/client').PrismaClient
+
+testMatrix.setupTestSuite(
+  ({ andQuery, orQuery, notQuery, noResultsQuery, badQuery }) => {
+    beforeAll(async () => {
+      await prisma.user.createMany({
+        data: [
+          {
+            email: 'email1@email.io',
+            name: 'John Smith',
+          },
+          {
+            email: 'email2@email.io',
+            name: 'April ONeal',
+          },
+          {
+            email: 'email3@email.io',
+            name: 'John Pearl',
+          },
+        ],
+      })
+    })
+
+    test('AND query', async () => {
+      const result = await prisma.user.findMany({
+        where: {
+          name: {
+            search: andQuery,
+          },
+        },
+      })
+
+      expect(result).toEqual([expect.objectContaining({ name: 'John Smith' })])
+    })
+
+    test('OR query', async () => {
+      const result = await prisma.user.findMany({
+        where: {
+          name: {
+            search: orQuery,
+          },
+        },
+      })
+
+      expect(result).toHaveLength(3)
+      expect(result).toContainEqual(expect.objectContaining({ name: 'John Smith' }))
+      expect(result).toContainEqual(expect.objectContaining({ name: 'April ONeal' }))
+      expect(result).toContainEqual(expect.objectContaining({ name: 'John Pearl' }))
+    })
+
+    test('NOT query', async () => {
+      const result = await prisma.user.findMany({
+        where: {
+          name: {
+            search: notQuery,
+          },
+        },
+      })
+
+      expect(result).toHaveLength(2)
+      expect(result).toContainEqual(expect.objectContaining({ name: 'April ONeal' }))
+      expect(result).toContainEqual(expect.objectContaining({ name: 'John Pearl' }))
+    })
+
+    test('no results', async () => {
+      const result = await prisma.user.findMany({
+        where: {
+          name: {
+            search: noResultsQuery,
+          },
+        },
+      })
+
+      expect(result).toEqual([])
+    })
+
+    testIf(process.platform !== 'win32')('bad query', async () => {
```

Comment:
Do we know why this is not run on windows?

Reply:
I think it runs on windows, but error message is very different from linux/mac and it is impossible to update snapshot without windows machine.